src/main/java/tornadofx/Rest.kt

```diff
@@ -187,12 +185,10 @@ open class Rest : Controller() {
             try {
                 val content = text()
 
-                if (content == null || content.isEmpty())
+                if (content.isNullOrBlank())
```

Comment:
This is a semantic change. Did you mean `isNullOrEmpty()`?

Reply:
In this case, you are right, I think the change is not justified. I will change it to `isNullOrEmpty()`